url-shortner-view/src/SecondPageView/SecondPageView.js

```diff
@@ -1,13 +1,17 @@
 import React from 'react';
 import SecondView from '../SecondView/SecondView';
-import classes from './SecondPageView.module.css';
-import Header from '../Header/Header';
-import ShareUrlSection from '../ShareUrlSection/ShareUrlSection';
 
-const SecondPageView = () => (
-	<main>
-		<SecondView />
-		<ShareUrlSection />
-	</main>
-);
+const SecondPageView = (props) => {
+  return (
+    <main>
+      <SecondView
+        url={props.url}
+        shortUrl={props.shortUrl}
+        setShortUrl={props.setShortUrl}
+        setUrl={props.setUrl}
+      />
+      <ShareUrlSection />
+    </main>
```

Comment:
look at the comment on View.js file and do the same please

Reply:
SecondPageView is displaying SecondView and ShareUrlSection not only SecondView